terraform/karpenter.tf

```
@@ -81,11 +81,11 @@ resource "kubectl_manifest" "karpenter_node_template" {
       name: default
     spec:
       subnetSelector:
-        karpenter.sh/discovery: ${module.eks.cluster_name}
+        karpenter.sh/discovery: "true"
       securityGroupSelector:
-        karpenter.sh/discovery: ${module.eks.cluster_name}
+        karpenter.sh/discovery: "spack-production"
```

```suggestion
        karpenter.sh/discovery: ${module.eks.cluster_name}
```
These are equivalent, while also avoiding hardcoding the cluster name in multiple places.